app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageFragment.kt

```diff
@@ -267,7 +271,10 @@ class NewMessageFragment : Fragment() {
             autoComplete = autoCompleteCc,
             onAutoCompletionToggledCallback = { hasOpened -> toggleAutoCompletion(CC, hasOpened) },
             onContactAddedCallback = { newMessageViewModel.addRecipientToField(it, CC) },
-            onContactRemovedCallback = { newMessageViewModel.removeRecipientFromField(it, CC) },
+            onContactRemovedCallback = {
+                newMessageViewModel.removeRecipientFromField(it, CC)
+                updateBannerVisibility()
+            },
```

Comment:
If it's as little complex and closely defined space-wise like in here, I don't think we need to bother with factorizing something like that. It would be more bothersome than useful. Nonetheless, here, by factorizing it I could make it just a slightly bit more readable so actually why not, I factorized it anyway